html/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="http://yui.yahooapis.com/3.18.1/build/cssreset/cssreset-min.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <div class="main">
+            <div class="flex-container">
+                <div class="flex-item grow1">
+                  <b>Checkout</b>  
+                </div>
+                <div class="flex-item grow2">
+                    Shipping  / <b>Payment Method</b>   /  Confirmation
+                </div>
```

Comment:
For separators in breadcrumbs ("/" character between navigation links ), please use “content” CSS Pseudo-elements as directed in the user story. Right now you put it using HTML use CSS for that.